src/views/Points/Locked.js

```diff
@@ -0,0 +1,192 @@
+import React, { useCallback, useMemo } from 'react';
```

Comment:
Isn't this file more or less just `views/Release/Locked`? We can deprecate/remove that one then, right?